demo/multi-release-jar-classpath/README.md

```diff
@@ -0,0 +1,36 @@
+# Building Multi-Release-Jar with Classpath Attributes
+
+This sample shows how to build a [Multi-Release-Jar](https://openjdk.org/jeps/238) with Tycho using the JDT classpath attribute approach.
+
+Instead of relying on the `Multi-Release: true` manifest header and special directory structure, this demo uses the Eclipse JDT's `multirelease` classpath attribute to mark source folders for specific Java releases.
```

Comment:
The attribute is not named `multirelease` it is named `release` this has to be fixed all over the place in this PR.